express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+const publicPath = path.join('__dirname', 'public');
+console.log(publicPath);
+
+const staticPublic = express.static('public');
+app.use(staticPublic);
```

Comment:
Now that you have your publicPath, you can use it as the argument to `express.static()`